lib/pbench/server/database/models/__init__.py

```diff
@@ -1,6 +1,8 @@
 import datetime
+from pyclbr import Class
```

Comment:
This is wrong. And while I thought I remembered using `type[Class]` in the past, I can't find any example and I have to suspect my memory was clouded. 😆 

Apparently the correct type hint is just `type`, as in `on_null: type`; and remove this import. Sorry. But the good news is that the change might even let your `decode_integrity_error` declaration fit on one line. (It knocks off 14 characters.)